### DISCUSSION TITLE
Error: Unable to download the mesh

### DISCUSSION THREAD

**Discussion by ederpardeiro** [amidaware/tacticalrmm#1947] [General]
Hi guys!

I'm having a problem to install agents. When I try to install, the message bellow apear:

![image](https://github.com/user-attachments/assets/c35fffea-12f4-49c6-9cc1-d30548a67686)

Anyone know how to solve this problem?

**Comment by wh1te909** [comment]
what TRMM version are you on? should have been fixed in 0.19.X

**Reply by ederpardeiro** [reply]
I already using the last version (v0.19.2), but even so forced update (./update.sh --force) and the error keeping appearing

**Reply by silversword411** [reply]
What happens when you go to the mesh url in browser from the agent?

**Reply by ederpardeiro** [reply]
Hi guys!!!

Sorry, but I forget to answer here... 

As I had not any configuration or important data, I just reinstalled everything and then I saw where I was making a mistake. I was changing the field bellow with a personalized information, and it was causing the problem.

![image](https://github.com/user-attachments/assets/078c62c9-26fe-4a51-bd8f-c8b42b9578f9)


Thank you very much for your help!!!!
